bin/install-version

```diff
@@ -17,12 +17,15 @@ if [ ! "$#" -eq 1 ]; then
 	exit 1
 fi
 
+INSTALL_LOCATION=$HOME/.minimesos/bin
```

Comment:
I'm not sure hiding script in hidden directory is a good idea. Should we drop it to current directory?